integrations.yml

```diff
@@ -257,3 +257,9 @@
   website: "https://enjoyfieldtrip.com/?utm_source=basecamp&utm_medium=extras"
   icon: "fieldtrip.png"
   category: 1
+-
+  name: "Instabug"
+  description: "Integrating Instabug with Basecamp helps your team receive all the details they need about issues reported by testers and users from your mobile app using our SDK. With this integration, all bugs and user feedback can be converted into issues with just a click!"
```

Comment:
Could you clarify this description and edit it down to a short phrase that pitches what Instabug does for Basecamp users? Use Basecamp terminology, too: we have to-dos, not issues.